src/party.cpp

```diff
@@ -419,7 +421,7 @@ void Party::shareExperience(uint64_t experience, Creature* source/* = nullptr*/)
 	leader->onGainSharedExperience(shareExperience, source);
 }
 
-bool Party::canUseSharedExperience(const Player* player) const
+bool Party::canUseSharedExperience(const Player* player)
```

Comment:
I don't like the way you're doing this tho, I would rather have a new ReturnValue than this and then use:

```cpp
ReturnValue returnValue = canEnableSharedExperience();
this->sharedExpEnabled = returnValue == RETURNVALUE_SHAREDENABLE ? true : false;
leader->sendTextMessage(MESSAGE_INFO_DESCR, getReturnMessage(returnValue));
```
Something like that.